test/ctia/task/migration/migrate_es_stores_test.clj

```diff
@@ -60,23 +60,24 @@
                   helpers/fixture-properties:clean
                   es-helpers/fixture-properties:es-store]))
 
-;; This a is a `defn` to prevent side-effects at compile time
-(defn es-props []
-  (p/get-in-global-properties [:ctia :store :es]))
+(defn es-props [get-in-config]
+  (get-in-config [:ctia :store :es]))
 
-;; This a is a `delay` to prevent side-effects at compile time
-(def es-conn
-  (delay
-    (connect (:default (es-props)))))
+(defn es-conn [get-in-config]
+  (connect (:default (es-props get-in-config))))
 
-;; This a is a `defn` to prevent side-effects at compile time
-(defn migration-index []
-  (p/get-in-global-properties [:ctia :migration :store :es :migration :indexname]))
+(defn migration-index [get-in-config]
+  (get-in-config [:ctia :migration :store :es :migration :indexname]))
 
 (defn fixture-clean-migration [t]
```

Comment:
FWIW an almost identical fixture lives in the other file in this PR, but refers to a different `es-conn`.